examples/subplots_axes_and_figures/figure_size_units.py

```diff
@@ -0,0 +1,78 @@
+"""
+==============================
+Figure size in different units
+==============================
+
+The native figure size unit in matplotlib is inches, deriving from print
+industry standards. However, users may need to specify their figures in other
+units like cm or pixels. This example illustrates how to do this efficiently.
```

Comment:
You should spell out centimetres, just like inches, though using `cm` for the scale factor is fine.
```suggestion
units like centimetres or pixels. This example illustrates how to do this efficiently.
```